hermes-console/src/components/navigation-drawer/NavigationDrawer.vue

```diff
@@ -0,0 +1,130 @@
+<script setup lang="ts">
+  import { isAdmin } from '@/utils/roles-util';
+  import { useAppConfigStore } from '@/store/app-config/useAppConfigStore';
+  import { useAuthStore } from '@/store/auth/useAuthStore';
+  import { useRoles } from '@/composables/roles/use-roles/useRoles';
+  import { useRoute } from 'vue-router';
+
+  import { computed, ref } from 'vue';
+  import NavigationItem from '@/components/navigation-drawer/NavigationItem.vue';
+  const authStore = useAuthStore();
+  const isLoggedIn = computed(() => authStore.isUserAuthorized);
+  const userData = computed(() => authStore.userData);
+  const route = useRoute();
+  const routeName = computed(() => route.name);
+
+  const configStore = useAppConfigStore();
+  const roles = useRoles(null, null)?.roles;
+  const rail = ref(false);
+</script>
+
+<template>
+  <v-navigation-drawer :rail="rail" permanent @click="rail = false">
+    <template v-if="!isLoggedIn">
+      <v-list>
+        <v-list-item
+          :prepend-avatar="`https://alleavatar.allegrogroup.com/${userData.user_name}.jpg`"
```

Comment:
Since it is opensourced repository we can't hardcode such internal links. We should remove it or make it a configurable string, but the second option would require more changes also on the backend site of hermes-management.

Reply:
I agree with Mateusz, can you remove this?